app/assets/javascripts/host_edit.js

```diff
@@ -173,7 +173,7 @@ function submit_with_all_params() {
       $('form input[type="submit"]').attr('disabled', false);
       if (window.location.pathname !== tfm.tools.foremanUrl('/hosts/new')) {
         // We got redirected to the show page, need to clear the title override
-        tfm.breadcrumbs.updateTitle();
+        tfm.store.dispatch('updateBreadcrumbTitle')
```

Comment:
nit: why removing semicolon if it was already there?

Also, should we delete webpack/assets/javascripts/foreman_breadcrumbs.js?